### ISSUE TITLE
Plans: main plans list remains focusable underneath plan details screen resulting in wrong information being read aloud

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-Android#10937]
Steps to reproduce:

1. Go to Settings > Accessibility > TalkBack > "Use service" or use the [Volume key shortcut](https://support.google.com/accessibility/android/answer/6006966?hl=en).
1. In the WordPress app, go to My Site > Plan.
1. Double tap any plan.
1. Open the global context menu by swiping down then right and double tap "Read from top".

**Result:** the wrong information is read aloud—you can hear the plan list screen being read even though you are on the plan details page. ([3m27s](https://cloudup.com/cCUsGLI_DJr))

<img src="https://user-images.githubusercontent.com/1119271/70564991-6854ac00-1b4e-11ea-89d8-5bcdeb6e9afc.png" width="270" alt="talkback-activity-detail-screen-reads-out-page-underneath-wpandroid-13 8-rc-1">
<sup>Tested with WPAndroid 13.8-rc-1 on Pixel 3 Android 10.</sup>

<p style="clear:both;"><sub>/hat tip @mchowning for the report, internal reference: p77Llu-c0a-p2</sub></p>

**Comment by designsimply** [comment]
Removing the `[Pri] High` label because even though this issue would be a frustrating experience in terms of accessibility and should be fixed, there is not actually a way to purchase plans via the app directly right now and so the flow is currently not a critical flow.

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
